platform/darwin/src/NSExpression+MGLAdditions.mm

```diff
@@ -60,11 +60,14 @@ + (void)installFunctions {
     INSTALL_METHOD(mgl_join:);
     INSTALL_METHOD(mgl_interpolate:withCurveType:parameters:stops:);
     INSTALL_METHOD(mgl_step:from:stops:);
+    INSTALL_METHOD(mgl_coalesce:);
     
     // Install functions that resemble control structures, taking arbitrary
     // numbers of arguments. Vararg aftermarket functions need to be declared
     // with an explicit and implicit first argument.
     INSTALL_CONTROL_STRUCTURE(MGL_LET);
+    INSTALL_CONTROL_STRUCTURE(MGL_MATCH);
+    INSTALL_CONTROL_STRUCTURE(MGL_SWITCH);
```

Comment:
I suggested `MGL_MATCH` and `MGL_SWITCH` as counterparts to `mgl_match:` and `mgl_case:`, respectively, but I think I got mixed up. `MGL_SWITCH` implies the same semantics as `MGL_MATCH`. I do think “case” would be problematic, as it makes it impossible to refer to the cases inside the expression. “Ternary” is also problematic because there can be multiple cases. Do you think `MGL_CONDITION` or `MGL_IF` could work?

Reply:
I think `MGL_IF` will do the work. It's more descriptive to what the function actually does.